contrib/packs/tests/test_action_unload.py

```diff
@@ -85,7 +85,7 @@ def test_run(self):
         config_dbs = Config.query(pack=pack)
 
         self.assertEqual(len(pack_dbs), 1)
-        self.assertEqual(len(action_dbs), 1)
+        self.assertEqual(len(action_dbs), 3)
```

Comment:
Wow! I agree that there are 3 actions in the `dummy_pack_1` pack. The other 2 actions were added back in May 2021. How long have these tests not been running correctly? Possibly since before the pack tests moved between GHA jobs. Yikes.

Reply:
> Wow! I agree that there are 3 actions in the `dummy_pack_1` pack. The other 2 actions were added back in May 2021. How long have these tests not been running correctly? Possibly since before the pack tests moved between GHA jobs. Yikes.

Hehe, I know ... what gap :smile: